pubsub/azure/eventhubs/eventhubs.go

```diff
@@ -558,7 +558,7 @@ func (aeh *AzureEventHubs) Publish(req *pubsub.PublishRequest) error {
 	if ok {
 		event.PartitionKey = &val
 	}
-	err := aeh.hubClients[req.Topic].Send(aeh.publishCtx, event)
+	err := aeh.hubClients[req.Topic].Send(ctx, event)
```

Comment:
The `publishCtx` property is now unused. Can you please delete that from the `AzureEventHubs` struct?

Reply:
Done.